nautobot/core/apps/__init__.py

```diff
@@ -652,3 +652,13 @@ class NautobotServerConfig(ConstanceConfig):
     """Override "Constance" app name to "Configuration"."""
 
     verbose_name = "Configuration"
+
+
+class ConstanceDatabaseAppConfig(AppConfig):
```

Comment:
Should this inherit from `constance.apps.ConstanceConfig` instead? There's some default behavior there (https://github.com/jazzband/django-constance/blob/master/constance/apps.py) that I'm not sure but what we may want to still retain.

Reply:
No I don't so as the issue is with `constance.backends.database` not `constance`

Reply:
Further to this, we already extend `constance.apps.ConstanceConfig` here: `nautobot.core.apps.NautobotServerConfig`. This is separate from `constance.backends.database` though.

Reply:
Good catch, I'd forgotten that these were two separate apps.